src/lib.rs

```diff
@@ -9,6 +13,7 @@ pub use winter_utils::{
     SliceReader,
 };
 
+pub use alloc::{format, vec};
```

Comment:
I would prefer to do it the same way we do in other places (unless there is a good reason to do it differently, and then we should update other places too):
```Rust
#[cfg(not(feature = "std"))]
pub use alloc::{format, vec};

#[cfg(feature = "std")]
pub use std::{format, vec};
```

Reply:
Fixed/simplified with the `macro_use`

I usually just include the requirements from either core or alloc because if std is available, they will be just a wrapper to it.

However, we can entirely remove this as in the solution above